src/devices/Ssd13xx/Commands/Ssd1327Commands/SetReMap.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Ssd13xx.Commands.Ssd1327Commands
+{
+    public class SetReMap : ISsd1327Command
+    {
+        /// <summary>
+        /// Re-map setting in Graphic Display Data RAM(GDDRAM) 
+        /// </summary>
+        public SetReMap(
+            bool columnAddressRemap = false,
+            bool nibbleRemap = true,
+            bool verticalMode = true,
+            bool COMRemap = false,
```

Comment:
casing should be `comRemap`. Please refer to https://github.com/dotnet/iot/blob/master/Documentation/Coding-guidelines.md